python/optional_requirements.yaml

```diff
@@ -21,6 +21,7 @@
 # To install ml4ir `all`, run pip install ml4ir[all]
 all:
   - pyspark==3.0.1  # required to run ml4ir.base.pipeline
+  - omnixai==1.1.4  #required for running explanations demo. Upgrade to 1.1.5 when it is available
```

Comment:
I meant to add it as another optional install too. So add something like this below.

```
explainer:
  - omnixai==1.1.4 #required for running explanations demo. Upgrade to 1.1.5 when it is available
```

Reply:
Done